Flipper/Packages/Core/Sources/Utils/CacheNetworkLoader.swift

```diff
@@ -0,0 +1,57 @@
+import Foundation
+
+actor GenericCacheNetworkLoader<Key: Hashable, Value> {
```

Comment:
It doesn't have network relation anymore. It's more like CachedTaskExecutor or something like that